docs/guide/install-and-run.asciidoc

```diff
@@ -0,0 +1,264 @@
+[[install-and-run]]
+== Install and run Elastic APM
+
+To get started using Elastic APM,
+you need to have:
+
+* an Elasticsearch cluster and Kibana (version 5.6 or above)
```

Comment:
whether this should theoretically work, we encourage users to have Es/Kibana/Server in the same version.